src/main/java/macrobase/analysis/pipeline/BasicBatchedPipeline.java

```diff
@@ -42,16 +43,14 @@ public Pipeline initialize(MacroBaseConf conf) throws Exception {
             outlierClassifier = new DumpClassifier(conf, outlierClassifier, queryName);
         }
 
-        MBOperatorChain<?, Summary> chain = MBOperatorChain.begin(data)
-                .then(new BatchScoreFeatureTransform(conf, conf.getTransformType()))
+        MBOperator<Datum, Summary> op =
```

Comment:
In cases like the main basic batched pipeline, what do you think about working with unwrapped operators and passing around their output directly? This will probably be easier to debug in the stack trace and dump debugging output from, while still being almost the same number of lines of code. 

One big advantage of the chaining syntax is that it allows us to wrap up multiple operators into one when we need it e.g. in the groupedby operator, but otherwise I thought the intermediate syntax of getStream.drain() was fine.